src/main/resources/META-INF/squbs-meta.conf

```diff
@@ -13,3 +13,8 @@ squbs-actors = [
     name = sample
   }
 ]
+squbs-extensions = [
```

Comment:
Reviewed the shutdown code. You should not need this. It is circular. Please let me know you need a discussion.